lib/auth/webauthncli/api.go

```diff
@@ -189,3 +214,15 @@ func Register(
 	}
 	return U2FRegister(ctx, origin, cc)
 }
+
+// HasPlatformSupport returns true if the platform supports client-side
+// WebAuthn-compatible logins.
+func HasPlatformSupport() bool {
+	return IsFIDO2Available() || touchid.IsAvailable()
+}
```

Comment:
@codingllama looking at it from outside, you cannot see that it can also be suypported from windows hello, maybe it should be after all something like that?
```suggestion
func HasPlatformSupport() bool {
	return isLibfido2Enabled() || webauthnwin.IsAvailable() || touchid.IsAvailable()
}
```

Reply:
I would go with the 1st, so eventual changes to IsFIDO2Available reflect automatically here.